### PR TITLE
Assistant selection in input bar: create / manage buttons

### PR DESCRIPTION
**Pull request by philipperolet**
From issue #1622 
Another PR adding a search bar will follow (issue #1721 )
image

### CODE REVIEW

File: front/components/assistant/conversation/InputBar.tsx [hunk 1]
```diff
@@ -609,6 +614,30 @@ export function AssistantInputBar({
                         }}
                       />
                     ))}
+                    <div className="flex flex-row justify-between border-t border-structure-100 px-3 py-2">
+                      <Button
```
Comment (fontanierh) [inline front/components/assistant/conversation/InputBar.tsx:618]: There is no way to use `<Link>` here ?

**Review by fontanierh — DISMISSED** [review]
Just a small question, but otherwise code lgtm so approving

**Review by Duncid — DISMISSED** [review]

File: front/components/assistant/conversation/InputBar.tsx (hunk 1 above)
Comment (spolu) [reply front/components/assistant/conversation/InputBar.tsx:618]: We can (and should) wrap the Button in a Link :+1:

File: front/components/assistant/conversation/InputBar.tsx [hunk 2]
```diff
@@ -609,6 +614,30 @@ export function AssistantInputBar({
                         }}
                       />
                     ))}
+                    <div className="flex flex-row justify-between border-t border-structure-100 px-3 py-2">
```
Comment (spolu) [inline front/components/assistant/conversation/InputBar.tsx:617]: Shouldn't this be a justify-center with a gap-x ?

Comment (philipperolet) [reply front/components/assistant/conversation/InputBar.tsx:617]: From the figma below

Comment (philipperolet) [reply front/components/assistant/conversation/InputBar.tsx:617]: image

Comment (philipperolet) [reply front/components/assistant/conversation/InputBar.tsx:617]: for me this is justify between and since ed approved I think we're good :)

**Comment by spolu** [comment]
These buttons should only be displayed to builder/admins otherwise we're sending users to 404?

**Pull request merged**